### PR TITLE
252 IB target/action

### LINKED ISSUE #252: BonMot disables IBActions dragging from storyboard
When Bonmot is included in my project, all my UIButton 'Touch Up Inside' events in the connections inspector via Utilities show warning signs. I suspect it has something with this stack overflow issue: http://stackoverflow.com/questions/28732903/cant-connect-ibaction-in-xcode.

screen shot 2016-11-26 at 12 21 13 pm

As one of the commenters suggested in the link, the culprit seems to be the UIButton extensions in BonMot's framework.

I commented the several extensions for UIButton out and now it works fine.

Going to work with clone bonmot into own project in meantime, but would love for this to be addressed!

### PR DESCRIPTION
**Pull request by ZevEisenberg**
Fixes #252.

- [ ] test to confirm that this works

### CODE REVIEW

File: Sources/UIKit/StyleableUIElement.swift [hunk 1]
```diff
@@ -8,20 +8,10 @@
...
-    var bonMotStyleName: String? { get set }
-
-    /// A string style. Stored via associated objects.
-    var bonMotStyle: StringStyle? { get set }
-
-    /// Update this property to style the incoming text via the `bonMotStyle`
-    /// and set it as the receiver's attributed text.
-    var styledText: String? { get set }
-
+/// Internal protocol to organize helper code for UI Elements. Originally this
```
Comment (ZevEisenberg) [inline Sources/UIKit/StyleableUIElement.swift]: #req describe what it does now, not what it used to do.

**Comment by ZevEisenberg** [comment]
@KingOfBrian this does not fix the problem as reported in #252, as far as I can tell.

**Comment by ZevEisenberg** [comment]
@KingOfBrian (at least, not in the sample app. Maybe that's an invalid test?)

**Comment by ZevEisenberg** [comment]
@KingOfBrian real-time followup: I made a new project and added BonMot pinned to cb46ad1425c3f6494801aecc1781b1664abe7814, and I couldn't connect an action to a UIButton in a storyboard, so I've confirmed that this fix is invalid.

**Comment by benyboariu** [comment]
Still no update on this?

**Comment by ZevEisenberg** [comment]
@benyboariu sorry, I've been out of the office. I'm going to do some more digging and see what I can get working.

**Comment by ZevEisenberg** [comment]
Superseded by #259.

**Pull request closed without merging**